spec/features/cco/defence_request_management_spec.rb

```diff
@@ -36,12 +36,15 @@
 
       specify "can acknowledge the request" do
         cco_user = create :cco_user
-        create :defence_request, :queued, cco_uid: cco_user.uid
+        dr =  create :defence_request, :queued, cco_uid: cco_user.uid
 
         login_with cco_user
         click_button "Acknowledge"
 
         expect(page).to have_content "Defence Request successfully acknowledged"
+
+        click_link "Show"
+        expect(page).to have_content "#{dr.dscc_number}"
```

Comment:
It's tricky. In my opinion we should be testing it where we do the operations. The `defence_request` model already has the transition condition. But because we test the individual transitions in isolation, I think it should be tested there.